recipes-core/customizations-example/files/board-configuration.json

```diff
@@ -1,28 +1,32 @@
 {
-    "User_configuration": {
+    "External_serial": {
```

Comment:
What does `External Serial` mean? the DB9?

The word `external` sounds vague to me, is the arduino uart serial external or internal?

Reply:
> What does `External Serial` mean? the DB9?
> 
> The word `external` sounds vague to me, is the arduino uart serial external or internal?

at here, it point to the DB9 not to the arduino.

Reply:
Then better rename it to something like `DB9_connector`

Reply:
ok, change the name to `External_db9_serial`